src/main/scala/Day2.sc

```diff
@@ -0,0 +1,71 @@
+import scala.io.Source
+
+//PATTERN MATCHING 1
+def booleanMathPattern(int1:Int, int2:Int, bool:Boolean) = {
+  val tup = new Tuple3(int1, int2, bool)
+  tup match {
```

Comment:
**val tup = new Tuple3(int1, int2, bool)**
and **tup match**
can be converted into (int1, int2, bool) match {